angular/projects/spark-angular/src/lib/components/sprk-dictionary/sprk-dictionary.component.spec.ts

```diff
@@ -35,4 +35,80 @@ describe('SprkAngularDictionaryComponent', () => {
     fixture.detectChanges();
     expect(element.getAttribute('data-id')).toBeNull();
   });
+
+  it('should add additionalClasses', () => {
+    component.additionalClasses = 'spark design';
+    fixture.detectChanges();
+    expect(element.classList.toString()).toContain('spark');
+    expect(element.classList.toString()).toContain('design');
+  });
+
+  it('should add additionalKeysClasses', () => {
```

Comment:
hey @bobwise , the new format is elementAdditionalClasses. This needs to be updated